docs/layouts/shortcodes/generated/pulsar_client_configuration.html

```diff
@@ -100,7 +100,7 @@
         </tr>
         <tr>
             <td><h5>pulsar.client.memoryLimitBytes</h5></td>
-            <td style="word-wrap: break-word;">0</td>
```

Comment:
I think this change deserves at least a separate commit. Is this changing the behavior or correcting the docs?

Reply:
Yep